cmd/thanos/sidecar.go

```diff
@@ -71,6 +80,8 @@ func runSidecar(
 	reloader *reloader.Reloader,
 	comp component.Component,
 	conf sidecarConfig,
+	GRPCLogOpts []grpc_logging.Option,
```

Comment:
```suggestion
	GRPCLogOpts []grpc_logging.Option,
```

`grpc_logging` can we improve pkg naming, because underscoe in pkg is forbidden by style guide

Reply:
Well, this is something which is being used in the previous imports as well, so I am not sure if we should change the convention here.

This would be nice, if on another PR, we could change the ordering for all the other imports. 
WDYT?